<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SDL_Qt_RGBClass</class>
 <widget class="QWidget" name="SDL_Qt_RGBClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1797</width>
    <height>1081</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SDL_Qt_RGB</string>
  </property>
  <widget class="QLabel" name="labelVideo_0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>351</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: rgb(151, 210, 255); 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelVideo_1">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>351</width>
     <height>341</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget {
    background-color: rgb(255, 20, 165); 
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>630</y>
     <width>140</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_Width_1"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>FPS:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="pushBtnSelectFile_1">
      <property name="text">
       <string>选择播放文件</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_FPS_1"/>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_Height_1"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_Format_1"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_17">
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>630</y>
     <width>140</width>
     <height>129</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <widget class="QSpinBox" name="spinBox_Width_0"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Format:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Width:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>FPS:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="3">
     <widget class="QPushButton" name="pushBtnSelectFile_0">
      <property name="text">
       <string>选择播放文件</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QSpinBox" name="spinBox_FPS_0"/>
    </item>
    <item row="1" column="2">
     <widget class="QSpinBox" name="spinBox_Height_0"/>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="comboBox_Format_0"/>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Height:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sdl_qt_rgb.qrc"/>
 </resources>
 <connections/>
</ui>
